flaky TestConnectionPathValidation

```
 --- FAIL: TestConnectionPathValidation (0.10s)
    --- FAIL: TestConnectionPathValidation/NAT_rebinding (0.00s)
        controller.go:97: missing call(s) to *quic.MockPacker.AppendPacket(is anything, is anything, is anything, is anything) D:/a/quic-go/quic-go/connection_test.go:2948
        controller.go:97: aborting test due to missing call(s)

--- a/connection_test.go
+++ b/connection_test.go
@@ -2948,7 +2948,7 @@ func testConnectionPathValidation(t *testing.T, isNATRebinding bool) {
 	calls = append(calls,
 		tc.packer.EXPECT().AppendPacket(gomock.Any(), gomock.Any(), gomock.Any(), gomock.Any()).Return(
 			shortHeaderPacket{}, errNothingToPack,
-		),
+		).MaxTimes(1),
 	)
 	gomock.InOrder(calls...)
 	require.Equal(t, tc.remoteAddr, tc.conn.RemoteAddr())

